### PR TITLE
Add exif

### PR DESCRIPTION
**Pull request by ktk**

### CODE REVIEW

File: ontologies/exif.nq [hunk 1]
```diff
@@ -0,0 +1,891 @@
+<http://tsc.jeita.or.jp/avs/data/cp3451.pdf> <http://purl.org/dc/elements/1.1/format> "PDF"@en <http://www.w3.org/2003/12/exif/ns> .
+<http://tsc.jeita.or.jp/avs/data/cp3451.pdf> <http://purl.org/dc/elements/1.1/title> "JEITA CP-3451/EXIF 2.2 Specification"@en <http://www.w3.org/2003/12/exif/ns> .
```
Comment (tpluscode) [inline ontologies/exif.nq:2]: 🤔

**Review by tpluscode — APPROVED** [review]
The vocabulary appears to bring in some extra triples but nothing "harmful" I guess

File: ontologies/exif.nq (hunk 1 above)
Comment (ktk) [reply ontologies/exif.nq:2]: Yeah some ontologies really have weird stuff in there. It's related to the discussion in #72, we should add it like this. It's nquads so it's separated in that regard.

**Comment by ktk** [comment]
Deleting this one, couldn't get it properly merged.

**Pull request closed without merging**